Jellyfin.Api/Helpers/DynamicHlsHelper.cs

```diff
@@ -592,6 +601,11 @@ private string GetOutputVideoCodecProfile(StreamState state, string codec)
             {
                 profileString ??= "main";
             }
+
+            if (string.Equals(state.ActualOutputVideoCodec, "av1", StringComparison.OrdinalIgnoreCase))
```

Comment:
This should be merged into the previous condition.

Reply:
Done